src/components/Table.js

```diff
@@ -1,74 +1,114 @@
 import React from "react";
-import GreenMark from "./StatusMarks/GreenMark";
-import RedMark from "./StatusMarks/RedMark";
-import NeutralMark from "./StatusMarks/NeutralMark";
+import moment from "moment";
+import "moment/locale/ru";
+
+import HabitStatus from "./HabitStatus";
+
+import { DateUtils } from "../utils/date";
+
+moment.locale("ru");
 
 const Status = {
   DONE: "DONE",
   FAILED: "FAILED",
+  NEUTRAL: "NEUTRAL",
   NOT_SPECIFIED: "NOT_SPECIFIED"
 };
 
-export default function Table({ HABITS }) {
-  function getMonday(d) {
-    d = new Date(d);
-    let day = d.getDay(),
-      diff = d.getDate() - day + (day === 0 ? -6 : 1); // adjust when day is sunday
-    return new Date(d.setDate(diff));
+export default class Table extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      READING_HABIT: {
+        name: "Чтение",
+        measuringValue: "Каждый день",
+        stats: [
+          { status: Status.DONE },
+          { status: Status.FAILED },
+          { status: Status.FAILED },
+          { status: Status.DONE },
+          { status: Status.FAILED },
+          { status: "" },
+          { status: "" }
+        ]
+      },
+      GYM_HABIT: {
+        name: "Спортзал",
+        measuringValue: "3р/нед",
+        stats: [
+          { status: Status.DONE },
+          { status: Status.FAILED },
+          { status: Status.FAILED },
+          { status: Status.DONE },
+          { status: Status.NEUTRAL },
+          { status: Status.NOT_SPECIFIED },
+          { status: Status.NOT_SPECIFIED }
+        ]
+      },
+      EARLY_RISE_HABIT: {
+        name: "Ранний подъем",
+        measuringValue: "6:30",
+        stats: [
+          { status: Status.DONE },
+          { status: Status.FAILED },
+          { status: Status.FAILED },
+          { status: Status.FAILED },
+          { status: Status.FAILED },
+          { status: Status.NOT_SPECIFIED },
+          { status: Status.NOT_SPECIFIED }
+        ]
+      }
+    };
   }
 
-  function getDayFromMonday(num) {
-    const date = new Date();
-    return new Date(date.setDate(monday.getDate() + num));
-  }
-
-  const monday = getMonday(new Date());
-  const tuesday = getDayFromMonday(1);
-  const wednesday = getDayFromMonday(2);
-  const thursday = getDayFromMonday(3);
-  const friday = getDayFromMonday(4);
-  const saturday = getDayFromMonday(5);
-  const sunday = getDayFromMonday(6);
+  handleClickCell = e => {
+    const test = e.target;
+    console.log(test);
+  };
```

Comment:
I think this handler is not needed in this ticket (it's not used anyway), you'll start working on this in the next one

Reply:
Ok, it was deleted